app/code/Darkwoolf/Person/view/frontend/layout/darkwoolf_person_jsonresponse_index.xml

```diff
@@ -5,7 +5,7 @@
     <body>
         <referenceContainer name="content">
             <container name="darkwoolf.person.name.block">
-                <block class="Stanislavz\LearnRoutes\Block\CustomBlock"
+                <block class="Darkwoolf\Person\Block\CustomBlock"
```

Comment:
Why did you change the block? What is the type of response this controller returns?